solr/solrj-zookeeper/src/java/org/apache/solr/common/cloud/SolrZkClient.java

```diff
@@ -1048,6 +1125,11 @@ public void downloadFromZK(String zkPath, Path dir) throws IOException {
     ZkMaintenanceUtils.downloadFromZK(this, zkPath, dir);
   }
 
+  @FunctionalInterface
+  public interface IsClosed {
```

Comment:
Does this new interface need to be public?

Reply:
It is a part of the builder, so it should probably be public.